test/test_xpu.py

```diff
@@ -0,0 +1,55 @@
+# Owner(s): ["module: xpu"]
+
+import unittest
+
+import torch
+from torch.testing._internal.common_utils import TestCase, \
+    run_tests, IS_WINDOWS
+from torch.testing._internal.common_device_type import (
+    instantiate_device_type_tests,
+)
+
+# not torch.backends.xpu.is_avaliable
+@unittest.skipIf(IS_WINDOWS, "XPU operators testing is disabled")
+class TestXPUOperators(TestCase):
+    def test_tensor_factory(self):
```

Comment:
Suggest not to pack multiple tests into a single UT. Separate them and name them explicity with `test_empty` and `test_empty_strided`.